Sources/AWSLambdaRuntimeCore/LambdaContext.swift

```diff
@@ -16,6 +16,33 @@ import Dispatch
 import Logging
 import NIO
 
+// MARK: - InitializationContext
+
+extension Lambda {
+    /// Lambda runtime initialization context.
+    /// The Lambda runtime generates and passes the `InitializationContext` to the Lambda handler as an argument.
+    public final class InitializationContext {
```

Comment:
also considered `InitContext` but this feels more correct since the function is called `initialize`